### PR TITLE
Feature/java ap i end point test

### PR DESCRIPTION
**Pull request by jianmingtu**
# Description

This PR includes the following proposed change(s):

- when we call get file secure on ccd criminal,  REST ORDS might either return responding messages or the data without responding messages.  REST SQL needs to change:  add a new layer 'fileContent' to both 
prGetFile and prGetFileSecure and JAVA api needs to update code corespondingly.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: jag-ccd-application/src/main/java/ca/bc/gov/open/ccd/controllers/FileController.java [hunk 1]
```diff
@@ -28,8 +27,11 @@
 @Endpoint
 public class FileController {
 
-    @Value("${ccd.host}")
-    private String host = "https://127.0.0.1/";
+    @Value("${ccd.host}" + "civil/")
```
Comment (ekwanNTT) [inline jag-ccd-application/src/main/java/ca/bc/gov/open/ccd/controllers/FileController.java:30]: Is having 2 really necessary cant we just combine in the url building?

File: jag-ccd-application/src/main/java/ca/bc/gov/open/ccd/controllers/UserController.java [hunk 2]
```diff
@@ -81,7 +81,7 @@ public MapGuidToParticipantResponse mapGuidToParticipant(
             HttpEntity<MapGuidToParticipantResponse> resp =
                     restTemplate.exchange(
                             builder.toUriString(),
-                            HttpMethod.POST,
+                            HttpMethod.GET,
```
Comment (ekwanNTT) [inline jag-ccd-application/src/main/java/ca/bc/gov/open/ccd/controllers/UserController.java:84]: Why this change?

Comment (jianmingtu) [reply jag-ccd-application/src/main/java/ca/bc/gov/open/ccd/controllers/UserController.java]: changed it back to HttpMethod.POST and also updated the CCD ORDS.

File: jag-ccd-application/src/main/java/ca/bc/gov/open/ccd/controllers/FileController.java (hunk 1 above)
Comment (jianmingtu) [reply jag-ccd-application/src/main/java/ca/bc/gov/open/ccd/controllers/FileController.java]: changed and only one configured host now.  each endpoint will then have its special url host.

**Review by ekwanNTT — APPROVED** [review]
